rholang/src/main/scala/coop/rchain/rholang/interpreter/dispatch.scala

```diff
@@ -20,7 +20,10 @@ object Dispatch {
 
   // TODO: Make this function total
   def buildEnv(dataList: Seq[Seq[Channel]]): Env[Par] =
-    Env.makeEnv(dataList.flatten.map({ case Channel(Quote(p)) => p }): _*)
+    Env.makeEnv(dataList.flatten.map({
+      case Channel(Quote(p)) => p
+      case Channel(_)        => Par() // Should never happen
```

Comment:
if should never happen then maybe leave the partial function as it was and add suppres warning to it. if it ever fails we want it to fail here (with a huge exception), rather then swallowing the error

Reply:
It seems like suppress warnings is not available in Scala https://stackoverflow.com/a/3506583/2750819 ?